app/tray_icon.js

```diff
@@ -0,0 +1,96 @@
+const electron = require('electron')
+const path = require('path');
+
+const app = electron.app;
+const Menu = electron.Menu;
+const Tray = electron.Tray;
+
+let trayContextMenu = null;
+let tray = null;
+
+function createTrayIcon(mainWindow, messages) {
+
+  // TODO: for better desktop theme integration, it would be good to provide
```

Comment:
If you want to put a TODO in the code, please create an issue and link to it directly in the code next to the TODO.